.github/workflows/ci.yml

```diff
@@ -11,17 +11,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: ['lts/-1', 'lts/*', 'node']
```

Comment:
please no. Keep the version picked as we don't want to roll them automatically.

Reply:
Could you elaborate a bit more on why you want to fix versions?

Reply:
Btw: I just pinned the NodeJS versions in CI